resources/documentation/monitoring/node_exporter.rst

```diff
@@ -0,0 +1,102 @@
+Node_exporter
+=============
+
+The node_exporter is an official exporter from Prometheus, and can be found
+here: https://github.com/prometheus/node_exporter
+
+By default, the node_exporter runs under the port 9100.
+
+To access the metrics, either do:
+
+.. code-block:: text
+
+  curl http://localhost:9100/metrics
+
+or access it directly in a browser. However, using curl can be handy, because
+you can grep the output, and do other nice things with it.
+
+You should get something like this:
+
+.. code-block:: text
+
+  # HELP node_vmstat_pgpgin /proc/vmstat information field pgpgin.
+  # TYPE node_vmstat_pgpgin untyped
+  node_vmstat_pgpgin 2.25160698e+08
+  # HELP node_vmstat_pgpgout /proc/vmstat information field pgpgout.
+  # TYPE node_vmstat_pgpgout untyped
+  node_vmstat_pgpgout 1.18421998e+09
+  # HELP node_vmstat_pswpin /proc/vmstat information field pswpin.
+  # TYPE node_vmstat_pswpin untyped
+  node_vmstat_pswpin 47719
+  # HELP node_vmstat_pswpout /proc/vmstat information field pswpout.
+  # TYPE node_vmstat_pswpout untyped
+  node_vmstat_pswpout 532036
+  # HELP process_cpu_seconds_total Total user and system CPU time spent in seconds.
+  # TYPE process_cpu_seconds_total counter
+  process_cpu_seconds_total 20272.88
+  # HELP process_max_fds Maximum number of open file descriptors.
+  # TYPE process_max_fds gauge
+  process_max_fds 1024
+
+There are a lot of information given by node_exporter. It is a very heavy
+exporter, so that is why it is not running on the compute nodes.
```

Comment:
Is it a recommendation? (Anyone can run it on the compute nodes with the appropriate playbook).

Reply:
yes it is a recommendation, I will rephrase that